src/core/turn-order.ts

```diff
@@ -24,6 +24,7 @@ export function SetActivePlayers(
   let activePlayers: typeof ctx.activePlayers = {};
   let _prevActivePlayers: typeof ctx._prevActivePlayers = [];
   let _nextActivePlayers: ActivePlayersArg | null = null;
+  let _activePlayersMinMoves = {};
   let _activePlayersMoveLimit = {};
```

Comment:
For consistency, let’s rename this `_activePlayersMaxMoves` here and everywhere else.